ods_ci/libs/DataSciencePipelinesAPI.py

```diff
@@ -88,7 +88,6 @@ def login_and_wait_dsp_route(
 
         assert self.route != "", "Route must not be empty"
         print(f"Waiting for Data Science Pipeline route to be ready: {self.route}")
-        time.sleep(30)
```

Comment:
I think this sleep was necessary to avoid pipeline alerts firing because of the test querying a route that is not ready yet

Reply:
It is firing. It will depend on the cluster. Why firing the alert is a problem?

Reply:
If the alert fires in our test when using the API because we don't wait for the route to be available but doesn't fire when using the Dashboard UI I think we are introducing noise in the test results

Reply:
Added it back